docs/development/internal_api.rst

```diff
@@ -12,3 +20,73 @@ Internal API
    :undoc-members:
    :private-members:
    :show-inheritance:
+
+Extensions
+----------
+.. automodule:: blocks.extensions.predicates
+   :undoc-members:
+   :members:
+   :private-members:
+   :show-inheritance:
+
+Monitoring
+----------
+.. automodule:: blocks.monitoring.aggregation
+   :undoc-members:
+   :members:
+   :private-members:
+   :show-inheritance:
+
+.. automodule:: blocks.monitoring.evaluators
+   :undoc-members:
+   :members:
+   :private-members:
+   :show-inheritance:
+
+Search
+------
+.. automodule:: blocks.search
+   :undoc-members:
+   :members:
+   :private-members:
+   :show-inheritance:
+
+Utils
+-----
+.. automodule:: blocks.config_parser
```

Comment:
This docstring is rendered at http://blocks.readthedocs.org/en/latest/configuration.html already